### DISCUSSION TITLE
Ending 16.42.3 LTS Support

### DISCUSSION THREAD

**Discussion by arichornlover** [arichornlover/uYouEnhanced#26] [General]
16.42.3 was an ipa I make for users who want uYou 2.1 to run stable but that’s gonna be coming to an end now. 16.42.3 has gotten too old to the point where new app issues occur and such. And this version still gets “some” ui changes here and there but the version is old. So this version of the YouTube App has a ton of issues regarding its features which can be only be fixed in the latest version or a newer LTS release of the app. Not only that this will be gone this will effect 17.49.6’s LTS Release’s aswell whenever the new “Actual” version of uYou releases.

I will release the final version of 16.42.3-2.1 later. Stay tuned y’all. No more changes or Improvements after this last update of 16.42.3 LTS.

**Comment by arichornlover** [comment]
the release of the final version is taking longer then I thought. I'm having some troubles building it which could mean something has changed.

**Reply by arichornlover** [reply]
F17 release has came out!, and also I finally fixed the annoying issue where OLED Dark Mode didn't work for the past 7 releases. at least I fixed it for y'all! and I'm really sorry for not fixing it sooner.

**Reply by arichornlover** [reply]
Also if y’all plan on building 16.42.3 then I highly recommend to don’t ever update iSponsorBlock to the latest version (v1.1.1) because it’ll crash and then the whole app probably won’t start up. 16.42.3 runs fine on iSponsorBlock v1.0-15 but just not on the latest due to the new changes in the code. And yes I did test it out to make sure things were working and that’s why I came to this discussion.

**Reply by arichornlover** [reply]
Alright I’ve had to make another release of 16.42.3-3.0 with the Version F18 Because I needed to add more changes to give the LTS version some more improvements. Updated LowContrastMode and it works on all of the UI elements now. And added a Version Spoofer in case yall want to use the older versions of 16.xx.x that includes that can bring back the old comment section, old description menu & channel menu.

**Reply by arichornlover** [reply]
Might need to do an F19 patch just to fix the Comment Section and the Playlists Dark Theme. I’m doing this patch to prevent and remove the new dark theme elements and may be necessary in some cases.

**Reply by arichornlover** [reply]
Some of the functionality on the app has stopped working
**Free YouTube TV Streams on YouTube**
![IMG_1237]
As you can see, you are no longer able to view Streams for free related to YouTube TV. Feature stopped working recently this year which means updating the app is the only fix.

**Channel “Sign Up” button on YouTube**
![IMG_1238](https://github.com/arichorn/uYouPlusExtra/assets/78001398/56c039ee-56e6-4843-8921-d68fdf46f078)
Unfortunately this feature stopped working. This feature does not matter if you’re sideloaded but instead if you are jailbroken it may be important to you if you had used it. Anyways the only way to use it again use newer YouTube app releases

**YouTube Search Menu Limited**
If anyone uses this version then I also got some bad news. I noticed this as well but google got rid of a ton of features in search. This is because they are forcing people to upgrade the app to a newer version. If you don’t understand what am talking about the I’ll tell you. When searching, it usually something related to that channel like for example “Music Artist” under a channel or song albums under their channel in the search. This feature still exists but it’s no where to be seen in 16.xx.x anymore. It could probably be an API change but not sure.

I’m still using 16.xx.x just because of the old UI. But I will definitely update this comment just in to update y’all if Google plans to do the same to other features.

**Reply by arichornlover** [reply]
Also I’m gonna have to do a 16.42.3-F20 release whenever I got time because YouTube has introduced a new change in August 2023 to the channel homepage which now shows links under your channel description in the channel’s home tab and I have to immediately remove it because it doesn’t have the functionality and since it doesn’t have that then the whole app will crash instantly and then corrupts YouTube completely if you tap on it. Right now please don’t press it I do a patch next release!

**Reply by arichornlover** [reply]
YouTube “tried” to add Community Posts to iPads on v16.42.3 recently this month. It’s so broken because it wasn’t even implemented in properly and the Community Posts Canvas Width wasn’t set. also pressing any post leads to the message `Community posts aren’t currently available on this device` which should be obvious since it wasn’t made for community posts on that version. Google needs to stop adding in new features in older yt versions because that can lead to breaking it.

**Comment by arichornlover** [comment]
F20 was the last release version of this LTS release. Hope everyone liked it. you can no longer download video/audio files when on 2.1 or 2.3~1. But you can use uYou Local.

**Comment by arichornlover** [comment]
![IMG_1425](https://github.com/arichorn/uYouEnhanced/assets/78001398/192f1114-91c8-4536-be7f-e68e8a2f4de9)
As of today, the original “Save to Playlist” Icon has been removed. I may need to reimplement the icon

**Comment by ego-lay-atman-bay** [comment]
Did youtube remove it?

**Reply by arichornlover** [reply]
Yep. It sucks that the old Save To Playlist Icon is gone. I could try to restore it if that’s possible.

**Reply by ego-lay-atman-bay** [reply]
I guess they can't stop removing things. First the dislike counter, now the save icon?

<details><summary></summary>
<p>

I'm assuming it's actually only for the old versions, not the current youtube app?

</p>
</details>

**Reply by arichornlover** [reply]
nope, if you go to `YouPiP` and then enable `Fake YouTube Version` play a video and you’ll see it aswell.

**Reply by ego-lay-atman-bay** [reply]
It's still the older version (given it would be spoofing youtube to think you're running an older version).

**Reply by arichornlover** [reply]
![IMG_2517](https://github.com/arichorn/uYouEnhanced/assets/78001398/39c4667b-d316-442e-b78b-699c8c75918d)
This might be the reason. I’m pretty certain the new logo on the left might be the reason behind this change?
